integration/networktest/runner.go

```diff
@@ -32,8 +32,8 @@ func Run(testName string, t *testing.T, env Environment, action Action) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	time.Sleep(45 * time.Second) // allow time for latest test transactions to propagate todo (@matt) consider how to configure this sleep
```

Comment:
isn't 45 seconds quite a lot?

Reply:
Oops yeah, was testing things out with this haha